db/citations.yml

```diff
@@ -7510,12 +7510,12 @@ liu;jacs21:
   optwwwlanl: ''
   pages: 2801-2811
   summary_prof_fig: ''
-  summary_prof_text: ''
+  summary_prof_text: Metal−organic materials offer a powerful platform for studying glass formation and crystallization processes and designing glasses with unique properties because of their high tunability and predictable structures. Here, we report a novel series of glass-forming metal−ethylenebis(acetamide) networks that undergo reversible glass and crystallization transitions below 200 $^\circ$C. The glass-transition temperatures, crystallization kinetics, and glass stability of these materials are readily tunable by synthetic modification or liquid-phase blending to form binary glasses. Pair distribution function (PDF) analysis reveals extended structural correlations in single and binary metal−bis-(acetamide) glasses. It highlights the critical role of metal−metal correlations during structural evolution across glass−crystal transitions. Notably, the glass and crystalline phases of a Co−ethylenebis(acetamide) binary network feature a large reflectivity contrast ratio of 4.8 that results from changes in the local coordination environment around Co centers. These results provide new insights into glass−crystal transitions in metal−organic materials and have exciting implications for optical switching, rewritable data storage, and functional glass ceramics.
```

Comment:
Metal−organic framework materials offer a powerful platform for studying glass formation and crystallization processes, as well as a platform for designing glasses with unique properties, because of their high tunability and predictable structures. Here, we report a novel series of glass-forming metal−ethylenebis(acetamide) networks that undergo reversible glass and crystallization transitions below the low temperature of 200 C. The glass-transition temperatures, crystallization kinetics, and glass stability of these materials are readily tunable by synthetic modification or liquid-phase blending to form binary glasses. Pair distribution function (PDF) analysis reveals extended structural correlations in single and binary metal−bis-(acetamide) glasses. It highlights the critical role of metal−metal correlations during structural evolution across glass−crystal transitions. 
These results provide new insights into glass−crystal transitions in metal−organic materials and have exciting implications for optical switching, rewritable data storage, and functional glass ceramics.